neuralop/layers/spectral_convolution.py

```diff
@@ -605,28 +569,28 @@ def forward(self, x, indices=0):
             slice(-self.n_modes[0] // 2, None),  # -half_n_modes[0]:,
             slice(self.n_modes[1]),  # ......      :half_n_modes[1]]
         )
-        print(f'2D: {x[slices0].shape=}, {self._get_weight(indices)[slices0].shape=}, {self._get_weight(indices).shape=}')
+        print(f'2D: {x[slices0].shape=}, {self.weight[slices0].shape=}, {self.weight.shape=}')
```

Comment:
For a future PR: for the 1D, 2D and 3D case, shall we make the indexing explicit rather than define these slices?

Reply:
That could be illustrative for doc purposes.